src/grid/grid-test.component.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
This should probably be in the spec file with other tests, added after imports but before tests start. That's the way other tests in the codebase approach this.

Reply:
Done